pkg/utils/utils_test.go

```diff
@@ -690,12 +688,6 @@ var _ = Describe("Utils", Label("utils"), func() {
 			Expect(err).ShouldNot(HaveOccurred())
 			Expect(size).To(Equal(int64(3072)))
 		})
-		It("Returns the expected size of a test folder", func() {
```

Comment:
why does this one need to be removed?

Reply:
This was using the non root path to somehow check for a failure getting the size of a dir, which is absurd. This is basically checking Linux permissions lol

Reply:
gotcha